junction/templates/proposals/partials/proposal-list--items.html

```diff
@@ -46,6 +46,14 @@ <h3 class="proposal--title">
                         <a class='tag tag__proposal_section label label-proposal-type inline-block'
                             href="?proposal_section={{proposal.proposal_section.id}}">{{ proposal.proposal_section }}
                         </a>
+                        <a class='tag tag__proposal_section label label-proposal-type inline-block'
+                            href="?proposal_status={{proposal.review_status}}">
+                            {% if proposal.review_status == 1 %}
+                                Reviewed
+                            {% elif proposal.review_status == 1 %}
```

Comment:
@theskumar Yes. Needs to handle the filtering too. Missed that. I thought earlier that it only needs to handle in the frontend to display.